### PR TITLE
Docker containers and configuration changes.

### PR DESCRIPTION
**Pull request by realARCHaim**

### CODE REVIEW

File: config/settings/base.py [hunk 1]
```diff
@@ -139,9 +136,18 @@
...
+
+# Default secure settings
+EMAIL_HOST = 'localhost'
 EMAIL_PORT = 25
 EMAIL_HOST_USER = ''
 EMAIL_HOST_PASSWORD = ''
 EMAIL_USE_TLS = False
-DEFAULT_FROM_EMAIL = 'AskUp mailer <mailer@askup.net>'
+
+SECRET_KEY = '(+wig4^c1749^^(23rp3$8y8r8_9o%-=#ujjfkasbjl=)-#b&o'
```
Comment (idegtiarov) [inline config/settings/base.py]: Please use default values from the secure_example.py

File: .dc-prod/django/Dockerfile [hunk 2]
```diff
@@ -2,15 +2,16 @@ FROM python:3.6
...
 RUN pip install -r requirements.txt
 
 VOLUME /var/log
-VOLUME /askup/askup/static
+VOLUME /askup
 
 EXPOSE 8001
 
-cmd until nc -w 3 postgres 5432; do sleep 1; done && /askup/init-prod-env.sh && python manage.py runserver 0.0.0.0:8001
+cmd until nc -w 3 postgres 5432; do echo "Trying to connect to the postgres...\n"; sleep 1; done && /askup/init-django.sh && while true; do python manage.py runserver 0.0.0.0:8001; sleep 5; done
```
Comment (idegtiarov) [inline .dc-prod/django/Dockerfile]: Please use uwsgi or gunicorn application server.

File: README.md [hunk 3]
```diff
@@ -1 +1,93 @@
-# askup
\ No newline at end of file
+# Bridge for Adaptivity
```
Comment (idegtiarov) [inline README.md]: Fill it up please and point me to continue review

**Review by idegtiarov — CHANGES_REQUESTED** [review]

File: config/settings/base.py (hunk 1 above)
Comment (realARCHaim) [reply config/settings/base.py]: done

File: .dc-prod/django/Dockerfile (hunk 2 above)
Comment (realARCHaim) [reply .dc-prod/django/Dockerfile]: done. now it uses the gunicorn in prod environment and runserver in a dev one.

File: README.md (hunk 3 above)
Comment (realARCHaim) [reply README.md]: done

**Review by idegtiarov — APPROVED** [review]
LGTM!

**Pull request merged**